cypress/integration/operation.spec.js

```diff
@@ -0,0 +1,98 @@
+describe('four-rule-calculations', () => {
+
+  const operandData1 = Array.from({length: 9}, (_, i) => i + 1) // 1~9 숫자
+  const operandData2 = [...Array(10).keys()] //0~9 숫자
+  const operator = ['+', '-', '/', 'X']
+  const operators = {
+    '+': (op1, op2) => op1 + op2,
+    '-': (op1, op2) => op1 - op2,
+    'X': (op1, op2) => op1 * op2, 
+    '/': (op1, op2) => Math.floor(op1 / op2)
+  };
+
+  beforeEach(() => {
+    cy.visit('http://localhost:5501')
+  })
+
+  
+  // 모든 한자리수 계산값을 구하는 과정이오니, 혹시 실행시켜보시는 분이 계신다면 주석처리 권장
+  operandData1.forEach(n1 => {
+    operandData2.forEach(n2 => {
+      operator.forEach(op => { 
+        it('한자리 숫자 두 개의 사칙연산', () => {
+          if (n2 === 0 && op == '/') {
+            return;
+          }
+          cy.get(`[data-cy=${n1}]`).click()
+          cy.get(`[data-cy='${op}']`).click()
+          cy.get(`[data-cy=${n2}]`).click()
+          cy.get(`[data-cy='=']`).click()
+          cy.get('#total').should(($total) => {
+            expect(parseInt($total.get(0).innerText)).to.eq(operators[op](n1, n2))
+          })
+        })
+      }) 
+    })
+  })
+
+  it('두 자리수 더하기', () => {
+    cy.get(`[data-cy=2]`).click()
+    cy.get(`[data-cy=5]`).click()
+    cy.get(`[data-cy='+']`).click()
+    cy.get(`[data-cy=1]`).click()
+    cy.get(`[data-cy=2]`).click()
+    cy.get(`[data-cy='=']`).click()
+    cy.get('#total').should(($total) => {
+      expect(parseInt($total.get(0).innerText)).to.eq(37)
```

Comment:
오 이렇게도 비교가 되는군요!! 참고하겠습니다~
저는  다르게 했는데 이런방법도 있더라구요 참고만 하세요 ㅎㅎ
```suggestion
      cy.get('#total').should('have.text', '37');
```

Reply:
인용님의 테스트코드 보면서 많이 배웠습니다 🙏🙏  
``cypress`` 사용법이 익숙하지 않아서 이런저런 방법으로 작성했는데, 말씀하신게 가독성이 더 뛰어나네요 참고하겠습니다 👍👍